### DISCUSSION TITLE
Beginners' FAQ in the Wiki

### DISCUSSION THREAD

**Discussion by peterhinch** [micropython/micropython#12514] [Using MicroPython]
The [Micropython Wiki](https://github.com/micropython/micropython/wiki) is an underused resource. I've made a start on a FAQ for people new to MP. Contributions are encouraged.

One oddity is that headings are rendered in blue so look like links. I've written a lot of markdown on Git and never come across this before. Does anyone know how to fix it?

**Comment by andrewleech** [comment]
Thanks @peterhinch I've really wanted to get more up there too, I applaud your efforts and yes I think we should all add stuff there!

The headings colour does stand out, yeah. They are "links" in the sense they have anchors that can be used as shortcuts and shared as links, but most other markdown renderers don't colour them as such. I can't see any suggestion of this being configurable at all, don't know if it can be changed at all.

**Reply by peterhinch** [reply]
Oddly other markdown docs on GitHub don't have this behaviour (bug?). It is possible to create links to normal headings, as in a TOC.

**Comment by Josverl** [comment]
> One oddity is that headings are rendered in blue so look like links. I've written a lot of markdown on Git and never come across this before. Does anyone know how to fix it?

Well, they **are** links, any reason to disable that?

**Reply by peterhinch** [reply]
That's odd: they are rendered in black this morning. It now looks and behaves like other Markdown on Git. I've no idea what changed overnight.

**Comment by Josverl** [comment]
Just noticed this popping up :
![image](https://github.com/micropython/micropython/assets/981654/8a4047e0-a1d6-4589-9706-e8852341e3ca)
links to: https://github.com/settings/accessibility

**Reply by peterhinch** [reply]
Thanks for that.

The behaviour of headings is still odd. Sometimes they are rendered in blue, sometimes in black. Yet the headings in other md docs are always black. It seems weird that the Wiki should be different. But it's hardly a showstopper :)
